### PR TITLE
feat: add rate_limit_weight into subscriptions and update rate-limit config

### LINKED ISSUE #51: cu config for subway proxy (using template from alchemy)
https://www.notion.so/alt-research/CU-table-alt-subway-ff8307541b9e410b99c120c0f6005981

### PR DESCRIPTION
**Pull request by koushiro**
Close #51

### CODE REVIEW

File: rpc_configs/geth.yml [hunk 1]
```diff
@@ -203,117 +235,162 @@ methods:
...
   # Web3 namespace (Geth-specified)
   - method: web3_clientVersion
+    rate_limit_weight: 15
     block_cache:
       size: 1
   - method: web3_sha3
+    rate_limit_weight: 15
 
 subscriptions:
   - subscribe: eth_subscribe
```
Comment (yjhmelody) [inline rpc_configs/geth.yml]: How about `subscribe` ?

File: rpc_configs/geth.yml [hunk 2]
```diff
@@ -34,31 +43,33 @@ methods:
...
-    params:
-      - name: Transaction
-        ty: Bytes
+#  - method: eth_sign
+#    params:
+#      - name: Address
+#        ty: String
+#      - name: Message
+#        ty: String
+#  - method: eth_signTransaction
```
Comment (yjhmelody) [inline rpc_configs/geth.yml:52]: These are disabled?

Comment (koushiro) [reply rpc_configs/geth.yml:52]: See https://www.notion.so/alt-research/CU-table-alt-subway-ff8307541b9e410b99c120c0f6005981 for details

File: rpc_configs/geth.yml (hunk 1 above)
Comment (koushiro) [reply rpc_configs/geth.yml]: I'll add it in next PR, because it will need to change code

Comment (koushiro) [reply rpc_configs/geth.yml]: #62

**Comment by koushiro** [comment]
cc @Revolution1

**Pull request merged**